pcsx2/gui/Panels/MemoryCardListView.cpp

```diff
@@ -162,9 +162,9 @@ wxString MemoryCardListView_Simple::OnGetItemText(long item, long column) const
 				//  it cannot access GetMcdProvider() which isn't (and shouldn't be) const.
 				//so.. a plain old cast does the trick. Hope it's not too bad. - avih.
 				if (((MemoryCardListView_Simple*)this)->GetMcdProvider().IsNonEmptyFilesystemCards())
-					return _("[-- Unused cards --]");
+					return _("[-- Disabled cards --]");
 				else
-					return _("[-- No unused cards --]");
+					return _("[-- No disabled cards --]");
```

Comment:
Doesn't unused seem better?

Reply:
I think we should just make it into *No Cards*, No unused or even no disabled looks confusing. If you delete all memory cards then it will say the same which make it technically wrong..

Reply:
It will also solve the problem of DPI scaling i am having, but it is fine on 100%.

Reply:
Examples:
![image](https://user-images.githubusercontent.com/24227051/92042437-7dc3d300-ed7a-11ea-8061-065eaef5549c.png)

![image](https://user-images.githubusercontent.com/24227051/92042464-903e0c80-ed7a-11ea-8976-bed7bd4166e7.png)

Also avoids a double negative.

Reply:
I changed it back to avoid unicode problem